frontend/packages/dev-console/src/components/catalog/catalog-view/CatalogCategories.tsx

```diff
@@ -20,12 +20,15 @@ const CatalogCategories: React.FC<CatalogCategoriesProp> = ({
 }) => {
   const activeTab = _.has(categories, selectedCategory);
 
-  const renderTabs = (category, selectedCategoryId) => {
+  const renderTabs = (
+    category: CatalogCategory & { numItems?: number },
+    selectedCategoryId: string,
```

Comment:
With console naming conventions, this would be

```suggestion
    selectedCategoryID: string,
```

Reply:
I just added the typing here, I can add a second commit to just rename this.